smoke-test/src/test/java/org/opennms/smoketest/dcb/DcbMultipleNodesIT.java

```diff
@@ -0,0 +1,220 @@
+    private static final String DCB_PASSWORD = "dcbpass";
+    private static final String DCB_SCRIPT_NAME = "test";
+    private static final String DCB_SVC_NAME = "DeviceConfig";
+
+    private static final int SSH_PORT = 2222;
+
+    private static final String FOREIGN_SOURCE = "SmokeTests";
+
+    private static final OpenNMSProfile OPENNMS_PROFILE = OpenNMSProfile.newBuilder()
+            .withFile("device-config/test.dcb", "etc/device-config/" + DCB_SCRIPT_NAME + ".dcb")
+            .withFile("device-config/SmokeTests.xml", "etc/foreign-sources/pending/SmokeTests.xml")
+            .build();
+
+    @ClassRule
+    public static final OpenNMSStack STACK = OpenNMSStack.withModel(StackModel.newBuilder()
+            .withOpenNMS(OPENNMS_PROFILE)
+            .build());
+
+    @ClassRule
+    public static final GenericContainer<?> TARGET_CONTAINER = new GenericContainer<>(
+            new ImageFromDockerfile().withDockerfileFromBuilder(builder -> builder.from("linuxserver/openssh-server")
+                    .run("apk add --update tftp-hpa")
+                    .build()))
+            .withEnv("PASSWORD_ACCESS", "true")
+            .withEnv("SUDO_ACCESS", "true")
+            .withEnv("USER_NAME", DCB_USERNAME)
+            .withEnv("USER_PASSWORD", DCB_PASSWORD)
+            .withExposedPorts(SSH_PORT)
+            .withNetwork(Network.SHARED)
+            .withNetworkAliases("target")
+            .withCreateContainerCmdModifier(TestContainerUtils::setGlobalMemAndCpuLimits);
+
+    private static String targetAddress;
+
+    private static OnmsNode localNode;
+    private static OnmsIpInterface localInterface;
+    private static OnmsMonitoredService localService;
+
+    private static RestClient restClient;
+
+    @BeforeClass
+    public static void setupClass() throws Exception {
+        targetAddress = getContainerInternalIpAddress(TARGET_CONTAINER);
+
+        restClient = new RestClient(STACK.opennms().getWebAddress());
+
+        final Requisition requisition = new Requisition();
+        requisition.setForeignSource(FOREIGN_SOURCE);
+
+        // Time prep for the schedule
+        LocalTime time = LocalTime.now().plusMinutes(2);
+        String schedule = String.format("0 %d %d ? * *", time.getMinute(), time.getHour());
+
+        // Node in default location for triggering backup from core
+        // TODO make a loop for creation of at least 1000 nodes
+        for (int i = 1; i < 101; i++) {
+            final List<RequisitionMonitoredService> monitoredServiceList = new ArrayList<>();
+            monitoredServiceList.add(new RequisitionMonitoredService(DCB_SVC_NAME));
+
+            final RequisitionInterface requisitionInterface = new RequisitionInterface();
+            requisitionInterface.setIpAddr(targetAddress);
+            requisitionInterface.setManaged(true);
+            requisitionInterface.setSnmpPrimary(PrimaryType.PRIMARY);
+            requisitionInterface.setMonitoredServices(monitoredServiceList);
```

Comment:
There is `RequisitionInterface.putMonitoredService` which spares you from creating a list. Alternatively there is `Collections.singletonList()` or `List.of()`.

Reply:
Not sure what you mean here as this is part of DCB test we have. However, in a new version of my test, it is changed.